docs/guides/connecting_to_your_data/datasource_configuration/sql_components/_tab_data_connector_example_configurations_runtime.mdx

```diff
@@ -33,29 +19,12 @@ datasource_config: dict = {
 
 For this example, you will be using the `RuntimeDataConnector` as your `class_name`.  This key/value entry will therefore look like:
 
-```python
-    "class_name": "RuntimeDataConnector",
+```python name="define data_connector class_name for configured sql datasource"
```

Comment:
These blocks are all referencing a ConfiguredAssetSqlDataConnector rather than a RuntimeDataConnector

Reply:
Should be resolved now.